tests/layer_tests/mo_python_api_tests/test_mo_convert_tf.py

```diff
@@ -660,6 +661,60 @@ def test_unnamed_saved_model_dir(self, ie_device, precision, ir_version, temp_di
         test_params = {'input_model': saved_model_dir, 'use_new_frontend': False}
         self._test_by_ref_graph(temp_dir, test_params, graph_ref, compare_tensor_names=False)
 
+    def test_zero_copy(self, ie_device, precision, ir_version, temp_dir):
+        import tensorflow as tf
+        from openvino.tools.mo import convert_model
+        from openvino.runtime import compile_model
+        class LayerModel(tf.Module):
+            def __init__(self):
+                super(LayerModel, self).__init__()
+                self.var1 = tf.Variable(np.random.rand(3).astype(np.float32), name='var1')
+                self.var2 = tf.Variable(np.random.rand(3).astype(np.float32), name='var2')
+
```

Comment:
Should we have another test case with constant?

Reply:
Yes, we can test, that we do not loose constant values from tf.Const. Added such test.